docker-compose.yml

```
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       args:
-        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-robot.git
+        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-turtlebot.git
```

```suggestion
        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-robot.git
```
Default value should be isar-robot